packages/frontend/src/components/Map/Markers/Classes/OpacityMarker/OpacityMarker.tsx

```diff
@@ -1,11 +1,10 @@
-import React, { useEffect, useState, useMemo, useRef } from 'react'
+import './OpacityMarker.css'
```

Comment:
For some reason is the .pulse animation not applied. Not for the markers or for the new report button